leetcode_arrays.js

```diff
@@ -0,0 +1,107 @@
+/***************** Max Consecutive Ones */
+/**
+ * @param {number[]} nums
+ * @return {number}
+ */
+var findMaxConsecutiveOnes = function(nums) {
+    var maxcon = 0;
+    var currentcon = 0;
+    for(i=0;i<nums.length;++i){
+        num = nums[i];
+        if(num == 1){
+            currentcon++;
+        } else {
+            currentcon = 0;
+        }
+        if(currentcon > maxcon)
+            maxcon = currentcon;
+    }
+    return maxcon;
+};
+
+/***************** Find Numbers With Even Digits */
+/**
+ * @param {number[]} nums
+ * @return {number}
+ */
+var findNumbers = function(nums) {
+    edn = 0;
+    for(let i=0;i<nums.length;i++){
+        num = nums[i];
+        digitsnum = String(num).length;
```

Comment:
do a space/runtime analysis of this solution and see if you can do better.   I think you can do digit calculation in constant time if you use a logarithm, though i wouldn't really expect anyone to come up with that in an interview